moor/lib/src/dsl/table.dart

```diff
@@ -150,9 +150,16 @@ class UseRowClass {
   /// used to map database rows to the desired row class.
   final String constructor;
 
+  /// Enables reverse mapping generation
+  ///
+  /// When this option is set, moor generates custom row class to companion
+  /// class mapping
+  final bool generateReverseMapping;
```

Comment:
Not sure if the line length works out, but I think we should expand the documentation here:

```suggestion
  /// Generate a companion constructor mapping the referenced [type] to a companion.
  ///
  /// When this option is set (it defaults to `false`), moor generates a constructor in the
  /// matching companion class of this table mapping an instance of [type] to a suitable
  /// companion.
  /// This can be useful when a custom data class should be used for inserts or updates.
  /// For this purpose, make it implement the `Insertable` interface, enable 
  /// [generateToCompanion] and use the generated companion in your `toColumns`
  /// implementation.
  /// When using this field, make sure that your existing data class contains a getter 
  /// (or field) for each column in the database.
  final bool generateToCompanion;
```